ext/xmlwriter/php_xmlwriter.stub.php

```diff
@@ -6,85 +6,85 @@ function xmlwriter_open_uri(string $uri): XMLWriter|false {}
+function xmlwriter_end_cdata(XMLWriter $writer): bool {}
 
-function xmlwriter_write_cdata(XMLWriter $xmlwriter, string $content): bool {}
+function xmlwriter_write_cdata(XMLWriter $writer, string $content): bool {}
 
-function xmlwriter_text(XMLWriter $xmlwriter, string $content): bool {}
+function xmlwriter_text(XMLWriter $writer, string $content): bool {}
 
-function xmlwriter_write_raw(XMLWriter $xmlwriter, string $content): bool {}
+function xmlwriter_write_raw(XMLWriter $writer, string $content): bool {}
 
-function xmlwriter_start_document(XMLWriter $xmlwriter, ?string $version = "1.0", ?string $encoding = null, ?string $standalone = null): bool {}
+function xmlwriter_start_document(XMLWriter $writer, ?string $version = "1.0", ?string $encoding = null, ?string $standalone = null): bool {}
 
-function xmlwriter_end_document(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_document(XMLWriter $writer): bool {}
 
-function xmlwriter_write_comment(XMLWriter $xmlwriter, string $content): bool {}
+function xmlwriter_write_comment(XMLWriter $writer, string $content): bool {}
 
-function xmlwriter_start_dtd(XMLWriter $xmlwriter, string $qualifiedName, ?string $publicId = null, ?string $systemId = null): bool {}
+function xmlwriter_start_dtd(XMLWriter $writer, string $qualifiedName, ?string $publicId = null, ?string $systemId = null): bool {}
 
-function xmlwriter_end_dtd(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_dtd(XMLWriter $writer): bool {}
 
-function xmlwriter_write_dtd(XMLWriter $xmlwriter, string $name, ?string $publicId = null, ?string $systemId = null, ?string $subset = null): bool {}
+function xmlwriter_write_dtd(XMLWriter $writer, string $name, ?string $publicId = null, ?string $systemId = null, ?string $content = null): bool {}
 
-function xmlwriter_start_dtd_element(XMLWriter $xmlwriter, string $qualifiedName): bool {}
+function xmlwriter_start_dtd_element(XMLWriter $writer, string $qualifiedName): bool {}
 
-function xmlwriter_end_dtd_element(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_dtd_element(XMLWriter $writer): bool {}
 
-function xmlwriter_write_dtd_element(XMLWriter $xmlwriter, string $name, string $content): bool {}
+function xmlwriter_write_dtd_element(XMLWriter $writer, string $name, string $content): bool {}
 
-function xmlwriter_start_dtd_attlist(XMLWriter $xmlwriter, string $name): bool {}
+function xmlwriter_start_dtd_attlist(XMLWriter $writer, string $name): bool {}
 
-function xmlwriter_end_dtd_attlist(XMLWriter $xmlwriter): bool {}
+function xmlwriter_end_dtd_attlist(XMLWriter $writer): bool {}
 
-function xmlwriter_write_dtd_attlist(XMLWriter $xmlwriter, string $name, string $content): bool {}
+function xmlwriter_write_dtd_attlist(XMLWriter $writer, string $name, string $content): bool {}
 
-function xmlwriter_start_dtd_entity(XMLWriter $xmlwriter, string $name, bool $isparam): bool {}
+function xmlwriter_start_dtd_entity(XMLWriter $writer, string $name, bool $isParam): bool {}
```

Comment:
So far, we didn't use the `is` prefix. Even though this is not what I usually prefer, I think it would be nice to follow the current convention (or decide on using it everywhere, or deciding on making an exception here if it's really necessary).

Reply:
I think we really need to do this on a case-by-case basis. `$param` is very generic and benefits from disambiguation, most other cases do not.

Reply:
Good point! Then @dtakken please disregard my comment :)